src/client.h

```diff
@@ -252,6 +253,11 @@ struct ClientEvent
 			bool do_override;
 			float ratio_f;
 		} override_day_night_ratio;
+		struct{
```

Comment:
Hm, this is the last struct in a union of a lot of structs, and none of them have that space… the entire repo seems pretty well split on `struct{` vs `struct {`. Ok if I leave this consistent with the others for now?